hibike/lib/hibike/hibike_device.cpp

```diff
@@ -1,24 +1,24 @@
 #include "hibike_device.h"
 
 message_t hibikeBuff;
-uint64_t prevTime, currTime, heartbeatTime;
+uint64_t prevTime, currTime, heartbeatTime,sentHeartbeat, respHeartBeat;
 uint16_t params, old_params;
 uint32_t value;
-uint16_t subDelay;
+uint16_t subDelay,disableLatency;
```

Comment:
disableLatceny should not be a uint16.

There is arithmetic done between disableLatency and millis().  millis() returns an unsigned long, (uint32).  Please make sure that disableLatency is also an unsigned long.

In the current implimentation, there will be arithmetic errors after millis - resp heart beet exceeds 65535, otherwise known as 65 seconds.  Given that a match is 2:30 long (excluding disabled periods), it'd totally be feasible for a problem to occur here.